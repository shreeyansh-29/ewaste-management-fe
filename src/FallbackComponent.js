import React from "react";
import "./components/errorBoundary/errorBoundary.css"
import image from "./assets/images/something-went-wrong.png";
export const FallbackComponent = () => {
  return (
    <>
      <div className="row text-center column">
        <div className="col-lg-12 div1">
          <img src={image} alt="" className="img1" />
        </div>
        <div className="col-lg-12">
          <h1 className="error-h1">Oh no!</h1>
          <h2 className="error-h2">!!!!....Something went wrong....!!!!</h2>
          <h5 className="error-h5">Refresh or Try Again Later</h5>
        </div>
      </div>
    </>
  );
};
