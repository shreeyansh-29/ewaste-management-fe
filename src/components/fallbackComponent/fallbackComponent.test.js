/* eslint-disable no-undef */
import React from "react";
import Enzyme, {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import FallbackComponent from "./fallbackComponent";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

describe("should render FallBackComponent", () => {
  it("testing", () => {
    const wrapper = shallow(<FallbackComponent />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
