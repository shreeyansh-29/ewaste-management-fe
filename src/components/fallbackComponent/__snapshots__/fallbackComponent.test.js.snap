// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FallBackComponent testing 1`] = `
<Fragment>
  <div
    className="row text-center column"
  >
    <div
      className="col-lg-12 div1"
    >
      <img
        alt=""
        className="img1"
        src="something-went-wrong.png"
      />
    </div>
    <div
      className="col-lg-12"
    >
      <h1
        className="error-h1"
      >
        Oh no!
      </h1>
      <h2
        className="error-h2"
      >
        !!!!....Something went wrong....!!!!
      </h2>
      <h5
        className="error-h5"
      >
        Refresh or Try Again Later
      </h5>
    </div>
  </div>
</Fragment>
`;
